# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/sqwebmail/cimap-sharedfiles.diff
# Copyright (C) 2007 - 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- a/Makefile.am	2026-10-18 07:44:20.012184558 +0200
+++ b/Makefile.am	2026-10-18 07:44:20.296542945 +0200
@@ -24,8 +24,7 @@ EXTRA_DIST=INSTALL.html README NEWS READ
 	pgpkeys.txt config.rpath
 
 scriptbindir=$(sbindir)
-scriptbin_SCRIPTS= \
-	sharedindexinstall sharedindexsplit
+scriptbin_SCRIPTS=
 
 pkglibPROGRAMS=maildirmake deliverquota reformime makemime pcpd
 
